### DISCUSSION TITLE
Cannot encode individual tracks into individual encoded files

### DISCUSSION THREAD

**Discussion by rebop** [enzo1982/freac#331] [Help & Support]
I am trying to encode FLAC 24bit files to .m4a. Tried Apple Lossless and Core encoders. Checked and unchecked single file several time, however it will only create a single file rather than individual tracks. What am I doing wrong?

**Comment by enzo1982** [comment]
Are your source files one file per track or multiple tracks per file?

If the former, please check the output filename pattern. Depending on the files' metadata, it could be that your output filename pattern causes each track to generate the same output file name.

If the latter, do the individual tracks show up in the joblist after loading a file? If that's not the case, check whether reading chapters is enabled in the settings.

If that does not help, please give more info about your source files and used settings (output filename pattern in particular).

**Comment by rebop** [comment]
Thanks!

One file fper track. Here is the folder:

![image](https://user-images.githubusercontent.com/13785042/174686603-ebe23386-4249-4bb8-ad5b-ea363c7435fc.png)

And it creates ONE file. Not multiples with the same name. One big file.

And they all show up in the joblist - but it complains about some already ??? I want to say exists, but I cannot recall what it really said.

Trying again. This MUST be it. I wonder why It is using the Tittle rather than the Name.

![image](https://user-images.githubusercontent.com/13785042/174686872-47b22e43-8f85-4822-b774-508a87266ca6.png)


![image](https://user-images.githubusercontent.com/13785042/174686954-cf6ae5ae-d970-43b6-a819-6bb0adce5e2e.png)

**Comment by rebop** [comment]
OK, changed the Titles and cleaned things up and it worked. Only thing it did was to sort alphabetically rather than the order of the tunes in the folder. Must be a setting for that I missed.

**Comment by enzo1982** [comment]
The problem is that all your files have the same artist and title in the tags, but no track numbers. So with the usual output filename patterns, fre:ac will generate the same file name for each output file.

However, as your source file names contain the track number at the beginning, you should be able to get back the original sorting by setting the output filename pattern to `<filename>`. This will preserve the file names of the source files (except for the extension, of course).
